bokehjs/src/lib/models/annotations/box_annotation.ts

```diff
@@ -469,7 +470,8 @@ export class BoxAnnotationView extends AnnotationView implements Pannable, Pinch
   }
 
   get movable(): boolean {
-    return this.model.movable != "none"
+    const movable = this.model.movable != "none"
+    return this.model.is_handle ? movable : this.model.editable && movable
```

Comment:
So why did add `is_handle`?

In my first pass, I did not change this property accessor. But then, I noticed that on examples/basic/annotations/box_annotation.py, I was seeing the grab cursor on box annotations that were not actually movable. This made me realize that this property accessor should check both `editable` and `movable`:

```ts
get movable(): boolean {
  return this.editable && this.model.movable != "none"
}
```

But this caused the handles to not work correctly. At examples/interaction/tools/box_select_handles.py, the above change resulted in the cursor not updating correctly when hovering over the handles (it would remain a default cursor rather than change into a move or resize cursor).

I realized that the reason for this was that the box handles themselves were not instantiated with `editable: true`. So then I changed the common props above on line 134 to have `editable: true`. But then this created a new problem that I don't understand yet. The hover cursors over the handles were now correct, but now they wouldn't budge. I could not use them to resize the box. 

All I know so far is that it is somehow related to the `on_pan_start` method. Because if I replace `this.model.editable` with `this.model.is_handle` on line 510 below, I recreate the bug: the handles stay stuck in place.

Reply:
Adding `is_handle` is perfectly valid given the dual nature of box annotation being an annotation and its interactive handles. It would have been better if `BoxAnnotation` was split into more manageable components.